src/main/java/attendance/domain/PenaltyCrew.java

```diff
@@ -0,0 +1,61 @@
+package attendance.domain;
+
+import java.util.Objects;
+
+public class PenaltyCrew implements Comparable<PenaltyCrew> {
+
+    private final String name;
+    private final int absenceCount;
+    private final int lateCount;
+    private final Integer point;
+    private final AttendancePenalty attendanceStatus;
+
+    public PenaltyCrew(String name, int absenceCount, int lateCount) {
+        this.lateCount = lateCount;
+        this.absenceCount = absenceCount;
+        this.name = name;
+        this.point = absenceCount * 3 + lateCount;
+        this.attendanceStatus = AttendancePenalty.find(absenceCount, lateCount);
+    }
+
+    @Override
+    public int compareTo(PenaltyCrew o) {
+        int attendanceComparison = this.attendanceStatus.compareTo(o.attendanceStatus);
+        int pointComparison = this.point.compareTo(o.point);
+
+        if (attendanceComparison != 0) {
+            return attendanceComparison;
+        }
+        if (pointComparison != 0) {
+            return -pointComparison;
+        }
+        return name.compareTo(o.name);
+    }
```

Comment:
아래와 같이 Comparator 를 활용하면 정렬에 대한 가독성이 향상될 것 같아요!

```suggestion
    @Override
    public int compareTo(PenaltyCrew o) {
        return Comparator.comparing(PenaltyCrew::getAttendanceStatus)
                .thenComparing(PenaltyCrew::getPoint, Comparator.reverseOrder())
                .thenComparing(PenaltyCrew::getName)
                .compare(this, o);
    }
```

Reply:
Comparator를 함수형 인터페이스를 사용하여 리팩토링했습니다.